src/table/sticky-scrollbar/styles.scss

```diff
@@ -14,6 +14,10 @@
   inset-block-end: 0;
   inline-size: 100%;
 
+  &[data-stuck='false'] {
+    clip-path: inset(-9999px 0 0 0 round awsui.$border-radius-container);
```

Comment:
Instead of avoiding the top cropping with a the negative top coordinate, the border radius can be specified for only for the bottom corners:

```
inset(0 round 0 0 awsui.$border-radius-container awsui.$border-radius-container)
```

WDYT?

Reply:
That would generally work and in our currently supported browsers it would not make a visible difference. However, it is based on the assumption that a browser will never render anything above the area that we calculate for the scrollbar. If there is ever a bug in our calculation, or a browser introduces additional UI elements for their scroll interactions in the future (compare e.g. [scroll progress indicator bubbles on Android](https://i.sstatic.net/A9nfP.png)), this would limit the rendering area unnecessarily. With the negative top coordinate, we reduce the risk of clipping where we don't need it.